lib/utils/logger.js

```diff
@@ -251,7 +251,13 @@ function createDeviceLogger() {
 	// Return logging function
 	return (method, args, time) => {
 		logTime = time;
-		appLogger[method](...args);
+		if (appLogger[method]) {
+			appLogger[method](...args);
+		} else if (!appLogger[method] && appLogger[method.toLowerCase()]) {
+			appLogger[method.toLowerCase()](...args);
+		} else {
+			appLogger.log(...args);
```

Comment:
what about send sentry about not supported log type from IL?
@andriichumak is it have sense?

Reply:
sure, why not